code/modules/mob/living/simple_animal/abnormality/waw/hatred_queen.dm

```diff
@@ -57,15 +56,21 @@
 	var/counter_amount = 0
 	var/can_act = TRUE
 	var/teleport_cooldown
-	var/teleport_cooldown_time = 10 SECONDS
+	var/teleport_cooldown_time = 30 SECONDS
 	var/beam_cooldown
 	var/beam_cooldown_time = 30 SECONDS
+	var/beam_startup = 3.5 SECONDS
 	/// BLACK damage done in line each 0.5 second
 	var/beam_damage = 10
 	var/beam_maximum_ticks = 60
 	var/datum/looping_sound/qoh_beam/beamloop
 	var/datum/beam/current_beam
 	var/list/spawned_effects = list()
+	//hostile breach vars
+	var/hostile_breach = FALSE
```

Comment:
There's no need to dedicate an entirely new variable for it.
Checking for qliphoth counter is enough.